src/DistortableImageOverlay.js

```diff
@@ -274,6 +274,21 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
     return this;
   },
 
+  getAngle: function() {
+    var matrix = this._image.style.transform
+      .split('matrix3d')[1]
+      .slice(1, -1)
+      .split(',');
+
+    var angle = Math.round(Math.atan2(matrix[1], matrix[0]) * (180 / Math.PI));
```

Comment:
Any specific reason to round here? If not, lets just leave it unrounded?

```suggestion
    var angle = Math.atan2(matrix[1], matrix[0]) * (180 / Math.PI);
```